### ISSUE TITLE
Existence.java:43-45: Use this class to implement...

### ISSUE THREAD

**Issue by 0pdd** [jcabi/jcabi-github#1384]
The puzzle `1382-3526ef36` from #1382 has to be resolved:

https://github.com/jcabi/jcabi-github/blob/851fa039335aa886b99e58f066371bbb19ad9f16/src/main/java/com/jcabi/github/Existence.java#L43-L45

The puzzle was created by amihaiemil on 07-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
The job [#1384](https://github.com/jcabi/jcabi-github/issues/1384) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/1082045634); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by llorllale** [comment]
@0crat waiting on review for PR #1392

**Comment by 0crat** [comment]
> @0crat waiting on review for PR #1392 ([here](https://github.com/jcabi/jcabi-github/issues/1384#issuecomment-382830350))

@llorllale The impediment for [#1384](https://github.com/jcabi/jcabi-github/issues/1384) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `1382-3526ef36` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1384](https://github.com/jcabi/jcabi-github/issues/1384) is now out of scope
